Libplanet/Blockchain/BlockChain.MineBlock.cs

```diff
@@ -363,6 +363,20 @@ public partial class BlockChain<T>
                 }
             }
 
+            if (transactionsToMine.Count < Policy.MinTransactionsPerBlock)
+            {
+                if (stagedTransactions.Count > transactionsToMine.Count)
+                {
+                    throw new OperationCanceledException(
+                        "stagedTransactions higher than transactionsToMine.");
+                }
```

Comment:
Would we need to deal with this case separately?  IMHO, we don't need this special casing.

Reply:
Done. See the latest commit.